compiler/main/driver.cpp

```diff
@@ -454,6 +455,21 @@ static void setDevelSettings(const ArgumentState* state, const char* arg_unused)
   }
 }
 
+static void setCCFlags(const ArgumentState* state, const char* arg) {
```

Comment:
Will you please add a comment describing what this does (and why it's necessary)? I understand after looking at it, but was originally confused especially since ccflagsArgument also exists.